memegen/routes/image.py

```diff
@@ -19,11 +21,14 @@
 
 @blueprint.route("latest.jpg")
 def get_latest():
-    title = "Latest Meme"
-    try:
-        return display(title, app.image_service.latest)
-    except FileNotFoundError:
-        return display(title, "static/images/missing.png", mimetype='image/png')
+    kwargs = cache.get(0)
+
+    if not kwargs:
+        kwargs['key'] = 'custom'
+        kwargs['path'] = "welcome-to/memegen.link"
+        kwargs['alt'] = "https://github.com/jacebrowning/memegen/raw/master/memegen/static/images/missing.png"
```

Comment:
Use the GitHub CDN?

https://raw.githubusercontent.com/jacebrowning/memegen/master/memegen/static/images/missing.png